doc/nrf/libraries/others/date_time.rst

```diff
@@ -30,6 +30,11 @@ When this library retrieves the date-time information, it is fetched in the foll
 #. If the time information obtained from the nRF91 Series modem is not valid and the :kconfig:option:`CONFIG_DATE_TIME_NTP` option is set, the library requests time from an NTP server.
 #. If the NTP time request does not succeed, the library tries to request time information from a different NTP server, before it fails.
 
+If all time requests fail, and the :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` Kconfig option is greater than zero, the entire process listed above will be retried after a short interval defined by :kconfig:option:`CONFIG_DATE_TIME_RETRY_INTERVAL_SECONDS`.
+In that case, the :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` Kconfig option sets how many retries are attempted consecutively before giving up.
```

Comment:
In my opinion, `consecutively before giving up.` is important information, since any success will reset the retry counter. But otherwise, I have adopted these changes.

Reply:
It's not really needed, since once a request is successful, any new retries will be of requests after that. I don't think there are any common systems where number of retries would be tracked on a system level instead of an occurrence level.

However, your new "consecutive retries" wording reads much better, so we can use that.